### ISSUE TITLE
[Bug] rule 1011 fails for IF-ELSE one liner splitted

### ISSUE THREAD

**Issue by rikerfi** [MarketSquare/robotframework-robocop#798]
### What happened?

Bug details

```python
    IF  ${condition}  Log  hello  
    ...    ELSE  Log  Else-hello
```
Above fails whatsoever `...` location is.

### Operating System

Linux

### Robocop version

2.8.1

**Comment by mnojek** [comment]
@rikerfi I guess the problem you described is not about "failing" 1011 rule, but that it is reported and you expect it to not be reported on this piece of code, right?

Please try next time to clearly describe the issue and the expectations, e.g. by providing the command that led you to the issue and the expected outcome. That will make it easier to properly detect where the issue is and fix it :)

Anyway, if my assumptions are correct, I think the problem is not with the Robocop itself but with your code.
You tried to use inline IF, but [this is not how it is supposed to be used](http://robotframework.org/robotframework/latest/RobotFrameworkUserGuide.html#inline-if).
As the name suggests, inline IF should be contained in one line. If you split it, for example because of the line length constraint, it's better to use normal IF statement, instead of artificially using shorthand, but making it multiline. 
That is an antipattern and it makes it less readable and harder to understand.

Can you explain to me what is the reasoning behind writing it that way? Maybe I'm missing some context here.

I would advise doing it this way:

```
    IF    ${condition}
        Log    Hello
    ELSE
        Log    Different hello
    END
```

or just use proper inline IF:

```
    IF  ${condition}  Log  hello    ELSE  Log   Different hello
```

BTW This case gave me an idea for another INFO rule for improperly used inline IF.

**Comment by bhirsz** [comment]
I agree that the multiline inline IF would be abusing the syntax - at this point it's better to use block IF. However we can discuss if it should be reported as misaligned continuation block (which may be misleading) or some new rule like multiline-inlineif ?

**Comment by rikerfi** [comment]
I see your points. Actually, the problem came in our case when inline statement was 
```IF ${cond}  <very long statement> ELSE <another very long statement>``` 
which lead to either `line too long` or `misaligned continuation` violation.

RF user guide states:
```
Long inline IF structures can be split into multiple lines using the common ... continuation syntax,
but using a normal IF/ELSE structure or moving the logic into a test library is probably a better idea.
```
So, this is a bit troublesome. Maybe from Robocop user point of view `multiline-inlineif` makes sense.
What about if I want to configure `allow multiline-inlineif`?

**Comment by bhirsz** [comment]
Then you could disable such rule if it would exist (-e rule-name). And btw Robotidy automatically converts too long inline IFs to IF blocks

**Comment by mnojek** [comment]
I will create an issue for the new rule to detect multiline inline IFs.
I know they are syntactically correct, but they should be avoided at all cost. Not everything what's allowed, is good :)

And I agree that we need to suppress the `misaligned-continuation-row` rule for such cases.
